presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -35,8 +47,7 @@
     private boolean optimizeSingleDistinct = true;
     private boolean pushTableWriteThroughUnion = true;
 
-    private boolean columnarProcessing;
-    private boolean columnarProcessingDictionary;
+    private String processingOptimization = ProcessingOptimization.DISABLED;
```

Comment:
No idea. This is only refactor though and the patch does not change previous behavior.

If you think it should be changed I guess it would be nice to discuss it as an issue or create PR suggesting change (as it is really simple change to make)?